docs/source/nn.functional.rst

```diff
@@ -160,6 +160,12 @@ Non-linear activation functions
 .. autofunction:: hardtanh
 .. autofunction:: hardtanh_
 
+:hidden:`hardswish`
+~~~~~~~~~~~~~~~~~~
+
+.. autofunction:: hardswish
+.. autofunction:: hardswish_
```

Comment:
I'm getting module 'torch.nn.functional' has no attribute 'hardswish_' when I run "torch.nn.functional.hardswish_". I don't think it is an actual op